toolchain/check/import_ref.cpp

```diff
@@ -1970,6 +1970,7 @@ static auto TryResolveTypedInst(ImportRefResolver& resolver,
   auto virtual_functions =
       GetLocalInstBlockContents(resolver, import_vtable.virtual_functions_id);
 
+  auto specific_data = GetLocalSpecificData(resolver, inst.specific_id);
```

Comment:
```suggestion
  auto specific_data = GetLocalSpecificData(resolver, inst.specific_id);
  
```
(just consistent whitespace)

Reply:
I guess github is hiding what this diff is and/or I'm change blind (I looked at the local version in my git checkout too) - but I'll take it :)

Reply:
Oh, the newline, right... don't mind me. Apparently the clang-tidy bot doesn't like this new whitespace :/